corpusreaders/src/test/java/edu/illinois/cs/cogcomp/nlp/corpusreaders/TACReaderTest.java

```diff
@@ -0,0 +1,95 @@
+package edu.illinois.cs.cogcomp.nlp.corpusreaders;
+
+import edu.illinois.cs.cogcomp.core.datastructures.IntPair;
+import edu.illinois.cs.cogcomp.core.datastructures.textannotation.TextAnnotation;
+import edu.illinois.cs.cogcomp.core.datastructures.textannotation.XmlTextAnnotation;
+import edu.illinois.cs.cogcomp.core.utilities.StringTransformation;
+import edu.illinois.cs.cogcomp.core.utilities.XmlDocumentProcessor;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+/**
+ * Created by cddunca2 on 2/12/18.
```

Comment:
Add a note indicating why this is not a unit test, and pointing out the hard-coded path...